data/trie/trieStorageManager.go

```diff
@@ -181,13 +181,16 @@ func (tsm *trieStorageManager) Prune(rootHash []byte, identifier data.TriePrunin
 	rootHash = append(rootHash, byte(identifier))
 
 	if tsm.snapshotInProgress > 0 {
-		if identifier == data.NewRoot {
-			// TODO refactor pruning mechanism so that pruning will be done on rollback
-			// even if there is a snapshot in progress
-			return
-		}
-
-		tsm.pruningBuffer.add(rootHash)
+		//if identifier == data.NewRoot {
```

Comment:
Do you want to keep this commented code?

Reply:
No, I only needed the TODO. Changed.